lib-ee/emqx_ee_bridge/src/kafka/emqx_bridge_impl_kafka_producer.erl

```diff
@@ -293,12 +302,20 @@ make_client_id(BridgeName) ->
 
 %% Producer name must be an atom which will be used as a ETS table name for
 %% partition worker lookup.
-make_producer_name(BridgeName) when is_atom(BridgeName) ->
-    make_producer_name(atom_to_list(BridgeName));
-make_producer_name(BridgeName) ->
-    %% Woff needs atom for ets table name registration
-    %% The assumption here is bridge is not often re-created
-    binary_to_atom(iolist_to_binary(["kafka_producer_", BridgeName])).
```

Comment:
I think this one is fixed even for dry-runs.  I made a couple quick tests, and it seems to be the user-defined bridge name (without any random suffix).

Reply:
So was this fix after the base of my PR? I tested this and the bridge name was random...

Reply:
(I might have rebased just before pushing so not sure if the base was what I tested with...)

Reply:
I rebased on `release-50` and printed the BridgeName in this function, configured a bridge in the UI and pressed the test button three times and got: 
```
<<"bridges-create-dry-run-ef6ab73d">>
<<"bridges-create-dry-run-959ffc4e">>
<<"bridges-create-dry-run-0b915820">>
```
So I think this PR is still relevant.